### PR TITLE
Update team for 2024

### PR DESCRIPTION
**Pull request by timesth**
Changed Co-op Soc team for 2024

### CODE REVIEW

File: data/TeamData.js [hunk 1]
```diff
@@ -817,64 +829,88 @@ const MEMBERS = [
...
-      //         role: "",
-      //       },
-      //     ],
-      //   ],
-      // },
+      {
+        sectionName: "First Year Representatives",
+        members: [
+          [
+            //Get Tony's Pic
```
Comment (scorpiontornado) [inline data/TeamData.js]: Don't need this comment

File: data/TeamData.js [hunk 2]
```diff
@@ -817,64 +829,88 @@ const MEMBERS = [
...
+              description: ``,
+              name: "Tony Wang",
+              role: "",
+            },
+            // {
+            //   image: Mariam_Image,
+            //   description: ``,
+            //   name: "First Last",
+            //   role: "",
+            // },
```
Comment (scorpiontornado) [inline data/TeamData.js]: Don't think this should be commented out

File: data/TeamData.js [hunk 3]
```diff
@@ -817,64 +829,88 @@ const MEMBERS = [
...
+          ],
+        ],
+      },
+    ],
+    subcoms: [
+      {
+        icons: [faClipboard],
+        name: "HR",
+        description: `The HR portfolio is committed to strengthening teamwork and cross-portfolio communication within the society. They are responsible for organizing internal events, to help shape our culture at Co-op Soc.`,
+        members: ["Stella Lin", "James Liao", "Aiden Ahmad", "Rashid Abuzarov"],
```
Comment (scorpiontornado) [inline data/TeamData.js:851]: Formatting - one per line. Make sure to set up eslint / formatter, ask @lhvy for help if needed

File: data/TeamData.js [hunk 4]
```diff
@@ -817,64 +829,88 @@ const MEMBERS = [
...
+          "Emma Kirkland",
+          "Sania Kulkarni",
+          "Alex Knijnik",
+        ],
+      },
+      {
+        icons: [faEdit],
+        name: "Publications",
+        description: `The Publications portfolio is responsible for continuing to grow the society's online presence and keeping scholars up to date, focusing on the blog posts for the Co-op Soc website and the Chicken Coop podcast.`,
+        members: ["Jamie Hilliam", "Jack Preller", "Declan Gormley"],
```
Comment (scorpiontornado) [inline data/TeamData.js:892]: Same - formatting

File: data/TeamData.js (hunk 2 above)
Comment (scorpiontornado) [reply data/TeamData.js]: Nvm she's now copres instead. Should delete rather than uncommenting out

**Review by scorpiontornado — CHANGES_REQUESTED** [review]
Looks good, just minor formatting issues

**Comment by timesth** [comment]
Fixed code based on feedback

**Comment by scorpiontornado** [comment]
Also be mindful of commit naming conventions, eg starting with a capital

**Comment by timesth** [comment]
Will keep it lowercase in future commits. Thanks Nick.

File: data/TeamData.js [hunk 5]
```diff
@@ -817,64 +829,81 @@ const MEMBERS = [
...
-      //         image: Name_Image,
-      //         description: ``,
-      //         name: "First Last",
-      //         role: "",
-      //       },
-      //     ],
-      //   ],
-      // },
+      {
+        sectionName: "First Year Representatives",
```
Comment (scorpiontornado) [inline data/TeamData.js]:
```suggestion
        sectionName: "First Year Representative",
```
Probably should update now that Mariam's changed roles

File: data/TeamData.js [hunk 6]
```diff
@@ -817,64 +829,81 @@ const MEMBERS = [
...
+          "Emma Kirkland",
+          "Sania Kulkarni",
+          "Alex Knijnik",
+        ],
+      },
+      {
+        icons: [faEdit],
+        name: "Publications",
+        description: `The Publications portfolio is responsible for continuing to grow the society's online presence and keeping scholars up to date, focusing on the blog posts for the Co-op Soc website and the Chicken Coop podcast.`,
+        members: ["Jamie Hilliam", "Jack Preller", "Declan Gormley"],
```
Comment (scorpiontornado) [inline data/TeamData.js:892]: Formatting

Comment (scorpiontornado) [reply data/TeamData.js:892]: This should really be picked up by eslint - should add to the CI/CD pipeline if not there already. Also, should just happen automatically with a formatter

Comment (scorpiontornado) [reply data/TeamData.js:892]: Same for HR

File: data/TeamData.js (hunk 3 above)
Comment (scorpiontornado) [reply data/TeamData.js:851]: Don't think this was done, I meant something like:
```js
members: [
  "Stella Lin",
  "James Liao",
  "Aiden Ahmad",
  "Rashid Abuzarov",
]
```

Should be automated by the linter/formatter. Let me know if the formatter chose to do it this way, but looks more like you just not setting it up yet.

**Review by scorpiontornado — DISMISSED** [review]
Still think there are some formatting issues, have you set up the linter/formatter? If not, we can probably help you in the next meeting.

**Comment by scorpiontornado** [comment]
> Will keep it lowercase in future commits. Thanks Nick.

Actually, it should be sentence case like you had in a few of them, like "Update team data after 2024 EGM" and "Change Rashid's surname" - first letter should be capitalised. Good use of imperative verbs by the way.

**Comment by timesth** [comment]
Hey Nick, ive made the formatting changes, not too sure how to set up eslint yet, hopefully we can discuss on Wednesday.

**Comment by lhvy** [comment]
Checked on my machine, formatting is fine. Good to merge.

**Review by lhvy — DISMISSED** [review]

**Review by lhvy — APPROVED** [review]

**Pull request merged**